packages/db/tests/config.js

```diff
@@ -0,0 +1,97 @@
+const test = require('ava');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire').noPreserveCache();
+
+const { getRequiredEnvVar, getEnvConnectionConfig } = require('../dist/config');
+
+const dbConnectionConfig = {
+  username: 'postgres',
+  password: 'password',
+  database: 'postgres',
+  host: 'localhost',
+};
+
+const secretsManagerStub = sinon.stub().returns({
```

Comment:
🐜 Do we actually need sinon here?

```suggestion
const secretsManagerStub = () => ({
```

Reply:
No, we don't: 

a8be1f79a